platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -14,12 +14,22 @@
         LoadingModal.closeEvent($rootScope, function(event) { that.show(); });
```

Comment:
As far as I understand after a talk with Marcin all the modifications here were made to prevent a modal from being opened multiple times. I think we should find the reason of such behavior rather than hack around it.